### ISSUE TITLE
Remove misleading TA properties in new examples

### ISSUE THREAD

**Issue by ivila** [linaro-swg/optee_examples#119]
From this [commit](https://github.com/linaro-swg/optee_examples/commit/20e5a612e6303820c1e41faf8aa265404b28de82), the misleading `gp.ta.version` and `gp.ta.description` was removed in `hotp`, `hello_world` and `random`.
![image](https://github.com/user-attachments/assets/95752d9b-b86e-4713-a489-ea7000d03645)

However, when we add new examples, those misleading extra properties was added again, for example, in the [commit of add secure_storage into examples](https://github.com/linaro-swg/optee_examples/commit/ae02c09a584054e055d9b3225856661c5f9dfc39#diff-c39ce60ce779a8caefe07c329ffdcab39ab37036a2db655170fc39dd8f727c7bR32)
![image](https://github.com/user-attachments/assets/053fbe37-0c12-4795-b4dc-aca8b2e30c6f)

Currently in the repo, there are 3 examples having properties of `gp.ta.version` and `gp.ta.description` (`secure_storage`, `acipher`, `aes`), should we remove them again?

@jforissier

**Comment by jbech-linaro** [comment]
> Currently in the repo, there are 3 examples having properties of `gp.ta.version` and `gp.ta.description` (`secure_storage`, `acipher`, `aes`), should we remove them again?

Yes, if they sneak in again, then let's remove them again. Hopefully future reviews will catch them.

**Comment by ivila** [comment]
> > Currently in the repo, there are 3 examples having properties of `gp.ta.version` and `gp.ta.description` (`secure_storage`, `acipher`, `aes`), should we remove them again?
> 
> Yes, if they sneak in again, then let's remove them again. Hopefully future reviews will catch them.

I just create a PR, do the same thing as the old commit do(same changes, same commit message), is it proper?

**Comment by jbech-linaro** [comment]
> I just create a PR,

Great, thanks!

> do the same thing as the old commit do(same changes, same commit message), is it proper?

We'll discuss that in the PR itself, thanks!
